Fix Metabase command syntax in mcp.md (#73779)
metabase/metabase: docs/ai/mcp.md

> In Claude Code, for example, you can run the following snippet, and Claude will handle the OAuth flow for you:

Author: ```suggestion
In the terminal, for example, you can run the following command.
```

```

> claude add --transport sse metabase https://{your-metabase-url}/api/mcp

Reviewer: `claude` is a terminal command, not a Claude command. So we'd need to update the line above too:

> In Claude Code, for example, you can run the following snippet, and Claude will handle the OAuth flow for you:

```

> Replacing {your-metabase-url} with your Metabase address.

Author: ```suggestion
Replacing {your-metabase-url} with your Metabase address.  Once added, Claude Code will handle the OAuth flow for you:
```
